src/reportengine/figure.py

```diff
@@ -91,7 +91,19 @@ def savefiglist(figures, paths, output):
             fig, suffix = fig
         else:
             suffix = str(i)
-        res.append(savefig(fig, paths=paths, output=output, suffix=suffix))
+        suffix = normalize_name(suffix)
+        p_base = [paths[i].relative_to(output) for i in range(len(paths))]
+        p_full = [
+            str(p.with_name('_'.join((p.stem, suffix)) + p.suffix)) for p in p_base
+        ]
+        ref = savefig(fig, paths=paths, output=output, suffix=suffix)
+        html = f"""<p class="half_page_width">
+                   <img src={p_full[0]}>
+                   <a href={p_full[0]}>.png</a> <a href={p_full[1]}>.pdf</a>
+                   </p>"""
+        if i % 2 == 1:
```

Comment:
What is this for? I don't see it making any difference, admittedly after changing the `p` to `div`.

Reply:
If I didn't add this in, then the first two images would be fine, but the third would be on the next line and float to the right, which would screw everything else up from then on